# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/sdrpp/hotfix-stdexcept.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- SDRPlusPlus-1.0.4/core/src/utils/networking.cpp.vanilla	2026-07-30 17:10:40.452400485 +0300
+++ SDRPlusPlus-1.0.4/core/src/utils/networking.cpp	2026-07-30 17:10:52.104125096 +0300
@@ -1,5 +1,6 @@
 #include <utils/networking.h>
 #include <assert.h>
+#include <stdexcept>
 
 namespace net {
 
@@ -475,4 +476,4 @@
         
         return Conn(new ConnClass(sock, raddr, true));
     }
-}
\ No newline at end of file
+}

